R/methods-epi_archive.R

```diff
@@ -422,11 +422,11 @@ epix_merge = function(x, y,
 #' Finally, this is simply a wrapper around the `slide()` method of the
 #'   `epi_archive` class, so if `x` is an `epi_archive` object, then:
 #'   ```
-#'   epix_slide(x, new_var = comp(old_var), n = 120)
+#'   epix_slide(x, new_var = comp(old_var), before = 120)
```

Comment:
Should this be `before = 119`?

Reply:
(same question regarding all the calls to epix_slide, including other instances of n = 120, instances of n = 3, any others)

Reply:
Think this + the below has been changed.